localgov_demo.install

```diff
@@ -19,59 +19,35 @@ function localgov_demo_install() {
 
   // Set demo front page.
   $system_site = \Drupal::configFactory()->getEditable('system.site');
-  $system_site->set('page.front', '/node/91');
+  $system_site->set('page.front', '/node/107');
```

Comment:
When I do a site install, node 107 is the The Broads (/national-parks-uk/broads). If NIDs are being maintained then this should be node 67.

Reply:
Looks like the node ids are not maintained. Every time we add or remove nodes, the node id for the home page changes. Odd that you got something different though. I will test again.

Reply:
@finn using an entity query to load by UUID would be safer...

Reply:
Turns out we can just use the relative path https://github.com/localgovdrupal/localgov_demo/pull/59/commits/1a8e91d6cbfc32013f0bda2f659d24cbc5de8108 :)